nemo/collections/nlp/data/datasets/sgd_dataset/data_processor.py

```diff
@@ -132,9 +146,27 @@ def get_dialog_examples(self, dataset):
             )
         dial_file = self.dial_files[(self._task_name, dataset)]
         logging.info(f"Loading dialogue examples from {dial_file}.")
+
+        if not os.path.exists(self.slots_relation_file):
+            logging.error(f"The processed dialogue file ({dial_file}) does not exist.")
         with open(dial_file, "rb") as f:
             dial_examples = np.load(f, allow_pickle=True)
             f.close()
+
+        if not os.path.exists(self.slots_relation_file):
+            logging.error(
+                f"Slots relation file {self.slots_relation_file} does not exist. It disables the carry-over mechanism of state tracker for switches between services."
+            )
+            slots_relation_list = {}
+        else:
```

Comment:
did you make this {} fix for Jenkins only? if so, let's throw an error and fix the test/files

Reply:
Not that, it makes the code to work with previous processed files too. As this file helps just the multi-domain dialogues. I would make it to throw error.

Reply:
It is also not used with baseline tracker.